_includes/scenarios/integrate/data-model-overview.md

```diff
@@ -239,6 +240,46 @@ With the information in this type you can clearly identify the Transport object
 | reference | `string`                          | The ID of the Transport object the Response was sent/received in. |
 
+## LocalNotification
+
+A LocalNotification contains the local metadata for a [Notification](#notification).
+
+| Name             | Type                                                  | Description                                                                                                                             |
+| ---------------- | ----------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------- |
+| id               | `string`                                              | {% include descr_id class="LocalNotification" prefix="NOT" %}                                                                           |
+| isOwn            | `boolean`                                             | `true` if you sent the Notification, `false` if you received it.                                                                        |
+| peer             | `string`                                              | The Identity that sent you the corresponding Notification/that you sent the Notification to.                                            |
+| createdAt        | `string`                                              | {% include descr_createdAt class="LocalNotification" %}                                                                                 |
+| status           | [`LocalNotificationStatus`](#localnotificationstatus) | The current status of the Notification. See [below](#localnotificationstatus) for a list of all possible values.                        |
+| content          | [`Notification`](#notification)                       | The actual Content object this LocalNotification defines the metadata for.                                                              |
+| source           | [`LocalNotificationSource`](#localnotificationsource) | Information about the Transport object with which the Notification came in/was sent.                                                    |
+| receivedByDevice | `string` \| `undefined`                               | The ID of the device that received the Notification. For reasons of consistence, the Notification can only be processed by this device. |
+
+### LocalNotificationStatus
+
+Depending on whether it is an incoming or an outgoing Notification, there can be different statuses.
+
+Sent
+: This is the only valid status on sender side. If a Notification is sent and, therefore, a LocalNotification is created, its status will be set to `Sent`.
+
+Open
+: Receiving a Notification, a LocalNotification is created with status `Open`.
+
+Completed
+: After receiving a Notification, its items will be processed internally. If all processes finish successfully, the status of the LocalNotification will be set to `Completed`.
```

Comment:
I'm a bit confused of the formatting. Why not only `Sent: This is the only...` and so on?

Reply:
![image](https://github.com/nmshd/documentation/assets/146972016/1304dfb8-6915-4214-869a-aaa5fbe8f117)

I copied the formatting from LocalRequestStatus, which results in a distinction between status and explanation. I suppose writing in in just one row would result in standard text.

Reply:
ah, I see